src/frontend/screens/Game/GamePage/index.tsx

```diff
@@ -605,7 +607,20 @@ export default React.memo(function GamePage(): JSX.Element | null {
     }
 
     if (isMoving) {
-      return `${t('status.moving')}`
+      if (file) {
+        return `${t('status.moving-files', 'Moving file')}  '${file}':  ${
+          percent ? `${percent}` : '...'
+        }`
+      }
+
+      if (percent) {
+        return `${t(
+          'status.moving',
+          'Moving Installation, please wait'
+        )}:  ${percent}`
```

Comment:
Might want to do this with a translation variable, I don't think every language is going to have this `<message>: <percent>` form

Reply:
Check now it that was what you meant.